bindings/python/jsonschema_rs.pyi

```diff
@@ -0,0 +1,69 @@
+from typing import Any, Optional, TypeVar, Union
+from collections.abc import Iterator
+
+_SchemaT = TypeVar('_SchemaT', bool, dict[str, Any])
+
+
+def is_valid(
+        schema: _SchemaT,
+        instance: Any,
+        draft: Optional[int] = None,
+        with_meta_schemas: Optional[bool] = None
+) -> bool:
+    pass
+
+def validate(
+        schema: _SchemaT,
+        instance: Any,
+        draft: Optional[int] = None,
+        with_meta_schemas: Optional[bool] = None
+) -> None:
+    pass
+
+def iter_errors(
+        schema: _SchemaT,
+        instance: Any,
+        draft: Optional[int] = None,
+        with_meta_schemas: Optional[bool] = None
+) -> Iterator[ValidationError]:
+    pass
+
+
+class JSONSchema:
+
+    def __init__(
+            self,
+            schema: _SchemaT,
+            draft: Optional[int] = None,
+            with_meta_schemas: Optional[bool] = None
+    ) -> None:
+        pass
+
+    @classmethod
+    def from_str(
+            cls,
+            schema: str,
+            draft: Optional[int] = None,
+            with_meta_schemas: Optional[bool] = None
+    ) -> 'JSONSchema':
+        pass
+
+    def is_valid(self, instance: Any) -> bool:
+        pass
+
+    def validate(self, instance: Any) -> None:
+        pass
+
+    def iter_errors(self, instance: Any) -> Iterator[ValidationError]:
+        pass
+
+
+class ValidationError(Exception):
+    message: str
+    schema_path: list[Union[str, int]]
```

Comment:
Should we use ‘List’ instead to support users running type checkers on older Python versions?

Reply:
As far as I remember, it is possible to use the last features, because the interpreter does not look at these files

Reply:
https://typing.readthedocs.io/en/latest/source/stubs.html#syntax

Reply:
It turns out we can even go to the new unions syntax

Reply:
Awesome! Thank you for clarification